internal/note.go

```diff
@@ -52,36 +52,55 @@ func (env *Env) NewRandomNoteCmd() *cli.Command {
 		Usage:       "Shows a random note",
 		Aliases:     []string{"r"},
 		Flags: []cli.Flag{
-			newQuoteFlag(asQuote),
+			newQuoteFlag(&asQuote),
 			&cli.IntFlag{Name: "length", Aliases: []string{"l"}, Destination: &lenLimit, Value: -1, Usage: "Limit note length to this value"},
 		},
 		Action: func(c *cli.Context) error {
 			note := env.getRandomNote(lenLimit)
+			bookName := ""
 			if asQuote {
 				book := Book{}
 				env.DB.Find(&book, note.BookID)
-				fmt.Println(fmt.Sprintf("%s - %s ", note.Text, book.Name))
-			} else {
-				fmt.Println(note.Text)
+				bookName = book.Name
 			}
+			fmt.Println(getFormattedNoteText(note, false, bookName))
```

Comment:
I was thinking about it but I am not sure how to avoid fetching book from db for every note (to get name). For `getRandomNote` it is not a problem but for `showNotes` it will just look bad. I will explore `gorm` package in more depth and come back with answers (and hopefully solution).

Reply:
@turbaszek I made some improvements. PTAL